doc/user-guide.md

```diff
@@ -842,6 +842,7 @@ When the operator is not running in a cluster, the Manager will return an error
 [event_filtering]:./user/event-filtering.md
 [controller_options]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/controller#Options
 [controller_godocs]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/controller
+controller-runtime-reconcile-godoc]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/reconcile#Reconciler
```

Comment:
```suggestion
[controller-runtime-reconcile-godoc]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/reconcile#Reconciler
```

Reply:
Yep... It's Monday :-). I saw that and fixed it.  Thanks!